docs/proxy.md

```diff
@@ -0,0 +1,16 @@
+# How to use KubeOne in a proxy environment
+
+Create a KubeOne configuration as usual and add the following section:
+
+```yaml
+proxy:
+  http_proxy: 'http://proxy.example.com'
```

Comment:
This doesn't match the `kubeone config print` output:
proxy:
 http: ''
 https: ''
 noProxy: ''

Reply:
indeed, this is pre-0.6 config format. it's time to forget about it once and for all.